### PR TITLE
Avoid ISPN8 dependency in JDG66

### PR DESCRIPTION
**Pull request by vjuranek**
Switch to ISPN 8.1 in ISPN 8.1 plugin and avoid ISPN8 dependency in JDG66

### CODE REVIEW

File: plugins/infinispan80/pom.xml [hunk 1]
```diff
@@ -116,7 +116,7 @@
             </property>
          </activation>
          <properties>
-            <version.infinispan>8.0.1.Final</version.infinispan>
+            <version.infinispan>8.1.0.Beta1</version.infinispan>
```
Comment (mcimbora) [inline plugins/infinispan80/pom.xml]: I've created infinispan81 plugin in https://github.com/radargun/radargun/pull/317 to keep isfinispan80 plugin on 8.0.x version line.

File: plugins/infinispan80/src/main/java/org/radargun/service/InfinispanEmbeddedContinuousQuery.java [hunk 2]
```diff
@@ -37,7 +36,7 @@ public void removeContinuousQuery(String cacheName, Object cqListener) {
         return service.getCache(cacheName);
     }
 
-    public static class IspnContinuousQueryListener implements org.infinispan.query.continuous.ContinuousQueryResultListener {
+    public static class IspnContinuousQueryListener implements org.infinispan.query.continuous.ContinuousQueryListener {
```
Comment (mcimbora) [inline plugins/infinispan80/src/main/java/org/radargun/service/InfinispanEmbeddedContinuousQuery.java]: Hmm which version did this API change take place in? If it was in 8.1.x, a new implementation should be created in separate plugin as per previous comment, wdyt?

File: plugins/infinispan80/pom.xml (hunk 1 above)
Comment (vjuranek) [inline plugins/infinispan80/pom.xml]: I didn't noticed it, will switch to ISPN 8.1 plugin once merged. Can someone review and merge it? Thanks!

File: plugins/infinispan80/src/main/java/org/radargun/service/InfinispanEmbeddedContinuousQuery.java (hunk 2 above)
Comment (vjuranek) [inline plugins/infinispan80/src/main/java/org/radargun/service/InfinispanEmbeddedContinuousQuery.java]: yes, in 8.1, will do once your PR is merged.

**Comment by rvansa** [comment]
@vjuranek This lonely PR wants your attention, and https://github.com/radargun/radargun/pull/283 depends on it.

**Comment by vjuranek** [comment]
@rvansa the changes which need to be dona in this PR depends on PR #317, it needs to be merged first.

**Comment by rvansa** [comment]
@vjuranek #317 was merged.

**Comment by vjuranek** [comment]
PR updated and ready for review.

**Comment by rvansa** [comment]
Thanks, merged.
I was about to complain about camelcase in `*Hotrod*` classes (since it is written "Hot Rod", it should be spelled as HotRod) but then I realized it was me who started to spell it wrong... So let's keep that consistent.

**Pull request merged**